src/supernova/circuit.rs

```diff
@@ -836,8 +836,8 @@ mod tests {
       &params2,
       ro_consts1,
       ro_consts2,
-      &expect!["9836"],
-      &expect!["10384"],
+      &expect!["9844"],
+      &expect!["10392"],
```

Comment:
It seems this needs a rebase, no? We're already at 9844 for SuperNova
https://github.com/lurk-lab/arecibo/blob/dev/benches/common/supernova/mod.rs#L31